### PR TITLE
ARTESCA-9911 // sudo required enabled by default

### PR DESCRIPTION
**Pull request by hervedombya**
**Component**:

**Context**: 

**Summary**:

**Acceptance criteria**: 

---

Closes: #ISSUE_NUMBER

### CODE REVIEW

File: ui/src/containers/NodeCreateForm.tsx [hunk 1]
```diff
@@ -186,46 +190,47 @@ const NodeCreateForm = () => {
...
+                        label={intl.formatMessage({
+                          id: 'control_plane',
+                        })}
+                        checked={values.control_plane}
+                        value={values.control_plane}
+                        onChange={handleChange('control_plane')}
+                        onBlur={handleOnBlur}
+                      />
+
                       <Checkbox
```
Comment (ChengYanJin) [inline ui/src/containers/NodeCreateForm.tsx:234]: Actually we'd like to hide only the infra roles for artesca instance. We should still keep the Roles of `Workload Plane` and `Control Plane`.

image

**Review by JBWatenbergScality — APPROVED** [review]
LGTM !

File: ui/src/containers/NodeCreateForm.tsx (hunk 1 above)
Comment (JBWatenbergScality) [inline ui/src/containers/NodeCreateForm.tsx:234]: No, this changed recently  😉

**Review by ChengYanJin — APPROVED** [review]

**Pull request merged**